subsys/caf/modules/buttons.c

```diff
@@ -319,7 +359,11 @@ static void scan_fn(struct k_work *work)
 		switch (state) {
 		case STATE_SCANNING:
 			state = STATE_ACTIVE;
-			err = callback_ctrl(true);
+			err = callback_ctrl(MASK_ALL);
+
+			if (!err) {
+				err = set_cols(MASK_ALL, true);
```

Comment:
Renamed to `enforce_pin_state` - output enforces pin state (i.e. does not allow for floating pings)

Reply:
I can change naming if it's not accepted here.

Reply:
It is misleading.
The best way would be to split it into two functions. I thought about it during first review but decided it's not worth it. Now I think it actually might be...

What it could be... Maybe
```
set_cols_as_input(x, y) {
   return set_cols(x,y);
}
set_cols_as_output(x) {
   return set_cols(x);
}
```

And there ambiguity will go away.